loveship/components/inputs/TimePicker.tsx

```diff
@@ -51,15 +51,25 @@ export class TimePicker extends React.Component<TimePickerProps, TimePickerState
             this.setState({ ...this.state, value: newValue });
         } else if (dayjs(newValue, this.getFormat(), true).isValid()) {
             const value = dayjs(newValue, this.getFormat(), true);
-
             this.props.onValueChange({ hours: value.hour(), minutes: value.minute() });
             this.setState({ ...this.state, value: newValue });
         } else {
             this.setState({ ...this.state, value: newValue });
         }
     }
 
-    handleBlur = () => {
+    onToggle = (value: boolean) => {
```

Comment:
Make BaseTimeTimePicker component into uui-components and move all component logic to there. Make it for 2 skins.